.github/workflows/integration-tests.yml

```diff
@@ -5,11 +5,13 @@ on:
     branches:

```

Comment:
We should then remove running integration tests on master branch, and to make sure everything works, require linear history options (everything rebased from master), in that way we only need to run tests on PR if they are required